modules/virt-creating-a-custom-image-boot-source-web.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-and-using-boot-sources.adoc
+
+[id="virt-creating-a-custom-disk-image-boot-source-web_{context}"]
+= Creating a custom boot source
+
+Sometimes you want to prepare a custom disk image, based on an existing disk image, for use as a boot source.
+
+This procedure guides you through:
+
+* Preparing a custom disk image.
+* Creating a boot source from the custom disk image.
+* Attaching the boot source to a custom template.
+
+.Prerequisites
+* You must have access to the web server with the operating system image you want to customize.
```

Comment:
```suggestion
```
You can customize locally stored images, access via web access is not required.

Reply:
Removed Prereq.